id,module,name,file,doc,markers,status,message,duration
functional_correctness_test_folder/csn/gpt-4.1/partial_docstr/test_snippet_94.py::test_case_0,functional_correctness_test_folder.csn.gpt-4.1.partial_docstr.test_snippet_94,test_case_0,functional_correctness_test_folder/csn/gpt-4.1/partial_docstr/test_snippet_94.py,,xfail,xfailed,,0.0002255830040667206
functional_correctness_test_folder/csn/gpt-4.1/partial_docstr/test_snippet_94.py::test_case_1,functional_correctness_test_folder.csn.gpt-4.1.partial_docstr.test_snippet_94,test_case_1,functional_correctness_test_folder/csn/gpt-4.1/partial_docstr/test_snippet_94.py,,,failed,AttributeError: 'NoneType' object has no attribute 'get',0.00010129099246114492
functional_correctness_test_folder/csn/gpt-4.1/partial_docstr/test_snippet_94.py::test_case_2,functional_correctness_test_folder.csn.gpt-4.1.partial_docstr.test_snippet_94,test_case_2,functional_correctness_test_folder/csn/gpt-4.1/partial_docstr/test_snippet_94.py,,,passed,,0.00036483301664702594
functional_correctness_test_folder/csn/gpt-4.1/partial_docstr/test_snippet_94.py::test_case_3,functional_correctness_test_folder.csn.gpt-4.1.partial_docstr.test_snippet_94,test_case_3,functional_correctness_test_folder/csn/gpt-4.1/partial_docstr/test_snippet_94.py,,,failed,AttributeError: 'NoneType' object has no attribute 'get',0.002570415992522612
